src/Components/App/App.js

```diff
@@ -33,14 +36,34 @@ class App extends Component {
       <>
       <article className='App'>
         <Header />
+        <form>
+          <input
+            className='search-input'
+            type='text'
+            placeholder="Search for a Movie"
+            value={this.state.searchInput}
+            onChange={(event) => this.handleChange(event)}
+          />
+        </form>
         {this.state.error && <h3>{this.state.error}</h3>}
-        {!this.state.error &&
+        {!this.state.error && this.state.matchingMovies &&
+            <Route exact path="/" render={() => {
+                return <Movies
+                  movieData={this.state.matchingMovies}
+                  movieSelected={this.state.movieSelected}
+                  handleClick={this.handleClick}
+                  matchingMovies={this.matchingMovies}/>
```

Comment:
I think we need one more piece of functionality here that will help with the display of the movie to help with the clearing out/going to home page.

I will pull this down and fiddle around with the ternary I have in mind.